app/scss/_menu.scss

```diff
@@ -0,0 +1,53 @@
+.menu {
+  display: flex;
+  grid-area: menu;
+  align-items: center;
+  justify-content: center;
+  padding: 0.5em 1em;
+  margin: 0 -1em;
+  background-color: $color-4-hex;
+
+  &__title {
+    display: none;
+  }
+
+  &__items {
+    display: flex;
+    text-align: center;
+
+    :last-child {
+      margin: 0;
+    }
```

Comment:
nice, there is one more trick which helps you save some lines of code:
Instead of setting `marggin: 0;` to the last child while all others have `margin:  0 1em  0 0;` , you can set the margin to all children, except the last one:
```
&__item:not(:last-child) {
  margin-bottom: em;
}
```